mitodl/open-discussions
# Added Rest APIs to add/remove moderators in a channel

#### What are the relevant tickets?
Fix #128

#### What's this PR do?
Adds REST API to add and remove moderators in a channel.

#### How should this be manually tested?
Login as a moderator, create a user and make sure that the user exists on reddit, try adding and removing this user as a moderator.
